packages/utils/src/typedEventEmitter.ts

```diff
@@ -1,22 +1,3 @@
-/*
-
-This file should not be exported from index to prevent missing dependency/polyfill error.
-EventEmitter is accessible in nodejs but requires polyfills in web builds.
```

Comment:
this is solved and re-export now doesn't matter?

Reply:
Yes, if tree shaking works it's not an issue.